### DISCUSSION TITLE
[DinoMod] Requests? Suggestions? Let's hear it!

### DISCUSSION THREAD

**Discussion by LyleSY** [CleverRaven/Cataclysm-DDA#49314] [Ideas]
The last year or so of DinoMod development has given us a bunch of new dinos, including tons of new zombie evolutions with more waiting in Dev, plus new NPCs and dino related missions and locations. What's missing? What would make DinoMod more fun and interesting for you?

**Comment by actual-nh** [comment]
I suspect people may need to see (including playtest) what all is currently going in, including in 0.F-dev.
